src/main/java/de/leonhard/storage/LightningBuilder.java

```diff
@@ -124,7 +128,7 @@ public LightningBuilder setDataType(@NonNull final DataType dataType) {
   // Create the objects of our FileTypes
   // ----------------------------------------------------------------------------------------------------
 
-  public Config createConfig() {
+  public @NotNull Config createConfig() {
```

Comment:
Are those annotation really needed here?

Reply:
Nullity refactor says it is.